src/GovUk.Education.ExploreEducationStatistics.Data.Services/ObservationService.cs

```diff
@@ -125,6 +125,7 @@ public IEnumerable<Observation> FindObservations(ObservationQueryContext query)
                     .Observation
                     .AsNoTracking()
                     .Include(o => o.FilterItems)
+                    .Include(o => o.Location)
```

Comment:
Just wondering if you also need the same line `.Include(o => o.Location)`  in method FindObservations(SubjectMetaQueryContext query) below on line 143 because that's what actually uses `ObservationPredicateBuilder` in this code.

Reply:
I'm thinking probably not.  The reason this one is here is that I am replacing the "manual setting of Locations" code that was in this method beforehand due to the funky SQL being produced.  The other method has no such Location setting code, so I'd assume it doesn't need the Locations retrieving in order to work.